qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -823,23 +823,23 @@ test_status_in_pool_dir()
     local description_pattern="$5"
     local service_pattern="$6"
 
-    local status_log=${TEMPDIR}/$(mkfname ${cluster}-${pool}-${image}.mirror_status)
-    CEPH_ARGS='' rbd --cluster ${cluster} mirror image status ${pool}/${image} |
-        tee ${status_log} >&2
-    grep "^  state: .*${state_pattern}" ${status_log} || return 1
-    grep "^  description: .*${description_pattern}" ${status_log} || return 1
+    local status=$(CEPH_ARGS='' rbd --cluster ${cluster} mirror image status \
```

Comment:
> * `local status` needs to be declared separately, otherwise potential errors from the command would be swallowed

Got it. Done.

> * not following why `CEPH_ARGS=''` is needed here but not on `rbd mirror pool status` command, suggest just getting rid of it

Looks like it's needed. It "unsets" the `CEPH_ARGS` set by rbd_mirror_helpers.sh.  Otherwise I see that `mirror image status` command does not show the `service` field in its output, and the test fails.
```
2024-02-20T19:53:10.894 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:++ MIRROR_POOL_MODE=image
2024-02-20T19:53:10.894 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:++ export 'CEPH_ARGS=--id mirror'

...

2024-02-20T20:01:04.718 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ local status
2024-02-20T20:01:04.719 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:++ rbd --cluster cluster1 mirror image status mirror/image-primary9
2024-02-20T20:01:04.995 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:rbd: failed to query services: (13) Permission denied
2024-02-20T20:01:05.006 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ status='image-primary9:
2024-02-20T20:01:05.006 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:  global_id:   04ac2e8f-f9b3-4d81-a169-a4fb5de4143a
2024-02-20T20:01:05.006 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:  state:       up+unknown
2024-02-20T20:01:05.006 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:  description: remote image is not primary
2024-02-20T20:01:05.006 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:  last_update: 2024-02-20 20:01:03
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:  peer_sites:
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:    name: cluster2
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:    state: up+unknown
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:    description: remote image is not primary
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:    last_update: 2024-02-20 20:01:03'
2024-02-20T20:01:05.007 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ echo image-primary9: global_id: 04ac2e8f-f9b3-4d81-a169-a4fb5de4143a state: up+unknown description: remote image is not primary last_update: 2024-02-20 20:01:03 peer_sites: name: cluster2 state: up+unknown description: remote image is not primary last_update: 2024-02-20 20:01:03
2024-02-20T20:01:05.008 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stdout:image-primary9: global_id: 04ac2e8f-f9b3-4d81-a169-a4fb5de4143a state: up+unknown description: remote image is not primary last_update: 2024-02-20 20:01:03 peer_sites: name: cluster2 state: up+unknown description: remote image is not primary last_update: 2024-02-20 20:01:03
2024-02-20T20:01:05.008 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ grep '^  state: .*up+unknown'
2024-02-20T20:01:05.008 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stdout:  state:       up+unknown
2024-02-20T20:01:05.009 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ grep '^  description: .*'
2024-02-20T20:01:05.009 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stdout:  description: remote image is not primary
2024-02-20T20:01:05.009 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ '[' -n '' ']'
2024-02-20T20:01:05.010 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ echo up+unknown
2024-02-20T20:01:05.010 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ grep '^up+'
2024-02-20T20:01:05.011 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stdout:up+unknown
2024-02-20T20:01:05.011 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ grep 'service: *mirror..* on '
2024-02-20T20:01:05.012 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ return 1
2024-02-20T20:01:05.013 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ return 1
2024-02-20T20:01:05.013 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ RET=1
2024-02-20T20:01:05.013 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ for pid in ${COMPARE_PIDS[@]}
2024-02-20T20:01:05.014 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ wait 36290
2024-02-20T20:01:05.014 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ RET=1
2024-02-20T20:01:05.014 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ (( RET != 0 ))
2024-02-20T20:01:05.014 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ echo 'compare_demoted_promoted_image failed'
2024-02-20T20:01:05.014 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stdout:compare_demoted_promoted_image failed
2024-02-20T20:01:05.016 INFO:tasks.workunit.cluster1.client.mirror.smithi157.stderr:+ exit 1
```

Snippet from http://qa-proxy.ceph.com/teuthology/rraja-2024-02-20_19:29:47-rbd:mirror-main-distro-default-smithi/7568961/teuthology.log